### ISSUE TITLE
Cannot click on Getting Started checkboxes with Voice Control

### ISSUE THREAD

**Issue by dexterleng** [microsoft/vscode#164778]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.73.0-insider c3eebaead9d3bf1cb961b91fca1b8af31990d328
- OS Version: macOS 12.6

Steps to Reproduce:

1. Open VSCode, click on the Getting Started walkthrough
2. Enable Voice Control with Hints
    - Able to focus on the options, but unable to click on the checkbox
    - ![CleanShot 2022-10-27 at 11 23 02](https://user-images.githubusercontent.com/34204380/198183647-fdfd2b83-9410-4e32-89c1-358a8300d3aa.png)
3. Inspecting with the Accessibility Inspector:
    - Looks like the checkbox is a `Group > AXStaticText` rather than a button:
    - ![CleanShot 2022-10-27 at 11 26 08](https://user-images.githubusercontent.com/34204380/198184098-7019f756-7988-4ed4-a8e4-fcfdca0d4001.png)
